src/plugins/vis_types/xy/public/expression_functions/xy_vis_fn.ts

```diff
@@ -278,7 +297,9 @@ export const visTypeXyVisFn = (): VisTypeXyExpressionFunctionDefinition => ({
         splitRow: args.splitRowDimension,
         splitColumn: args.splitColumnDimension,
       },
-    } as VisParams;
+      // ------------------------------------------------------------------------------------------------------------------
+    } as VisParams; /* @TODO: rewrite this `as VisParams` to real `VisParams` via changing accessor 
```

Comment:
All these TODOs are going to be handled in the next steps?

Reply:
as for real, I can remove those @todo , because I've completed mostly all of them )